json_schema/type/biomaterial/specimen_from_organism.json

```
@@ -115,6 +115,13 @@
             "type": "object",
             "$ref": "module/process/purchased_reagents.json",
             "user_friendly": "Purchased specimen"
+        },
+        "organ_donor": {
+            "description": "Was the specimen collected after extraction for organ transplantation?",
```

```suggestion
            "description": "Was the specimen collected after extraction for organ transplantation?",
```

Does it matter if collection happened before or after extraction?

```suggestion
            "description": "Was the specimen collected from an organ about to be transplanted to another organism?",
```

---

Hi Hannes, bionetworks are interested in recording this since despite that this is not specimen from a living organism (in most transplant cases) the tissue is kept alive and therefore the transcriptome quality of the sample is in much better condition that to a post-mortem sample.

Therefore, yes, it is important to record that when collection happened after extraction.